fleetx/core/engine/static_engine.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import sys
+
+import paddle
+import paddle.nn as nn
+
+from paddle.distributed import fleet
+from paddle.distributed.auto_parallel.engine import Engine
+
+sys.path.append("../../../")
+from fleetx.utils import logger
+from fleetx.data.sampler import Stack, Tuple
+from fleetx.core.engine.basic_engine import BasicEngine
+from fleetx.core.module.basic_module import BasicModule
+
+
+class StaticEngine(BasicEngine):
```

Comment:
Please rename StaticEngine to AutoEngine, because we will employ the conversion from the dynamic graph to the static graph to support both the dynamic and static graph.